File: composeApp/src/toDestroyJava/src/Enumerise/KindData.java
package Enumerise;

public enum KindData {

SER("SER", "Mcal"),
DM("DM", "100g DM"),
BW("BW", "kg"),
MW("BW", "kg MW"),
FENER("BW", "Mcal"),
FDESC("BW", "100g"),
NO("no",""), 
AMINO("",""), 
INGRED("",""),
PP("",""),
INDICAT("",""),
LIP("","")


			;
			


		private String name = "";
		private String unite = "";
	

		//Constructeur
	KindData(String name, String unit ){
		this.name = name;
this.unite=unit;
		}




		public String nameToString(){
		return name;
		}
		public String getUnit(){
		return unite;
		}
}

File: composeApp/src/commonMain/kotlin/fr/vetbrain/vetnutri_mp/Enumer/KindData.kt
package fr.vetbrain.vetnutri_mp.Enumer



enum class KindData(private val nom: String, private val unite: String) {
    SER("SER", "Mcal"),
    DM("DM", "100g DM"),
    BW("BW", "kg"),
    MW("BW", "kg MW"),
    FENER("BW", "Mcal"),
    FDESC("BW", "100g"),
    NO("no", ""),
    AMINO("", ""),
    INGRED("", ""),
    PP("", ""),
    INDICAT("", ""),
    LIP("", "");

    fun nameToString() = nom
    fun getUnit() = unite
}
